keyboards/foxlab/leaf80/rules.mk

```
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32a
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = bootloadHID
+
+# build options
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
+
+BACKLIGHT_ENABLE = no
+RGBLIGHT_ENABLE = no
+WS2812_DRIVER = i2c
```

Looks like you can omit this line, because the PCB apparently has nothing connected to C0 (SCL) and C1 (SDA) pins.

And you should add `LAYOUTS = tkl_ansi` to make keymaps from `layouts/community` usable with this PCB. If you choose to add ISO support, add `LAYOUTS = tkl_ansi tkl_iso` here instead.

```suggestion

LAYOUTS = tkl_ansi
```

---

Yep, there is no RGB at all according to this album from MxBlu: https://imgur.com/a/2C7wAqN